tests/microos/selfinstall.pm

```diff
@@ -44,6 +46,11 @@ sub run {
         eject_cd() unless ($no_cd || is_usb_boot);
     }
 
+    # Remove usb boot entry and ensure installed system boots from hard disk
+    if (is_ipmi and is_uefi_boot and is_usb_boot) {
+        remove_efiboot_entry(boot_entry => 'OpenQA-added-UEFI-USB-BOOT');
```

Comment:
Is this a necessary step on vh015? It wasn't needed before https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19063/. But for this PR, I did not verify it on vh015 because you were using it.  
Besides, on `kermit` machine, it is needed to set boot to newly installed sl micro, which is different from vh015, so efibootmgr is used again in combustion script to set boot to sl micro, see [this](https://github.com/alice-suse/os-autoinst-distri-opensuse/blob/4594a82d6eb75c4e7d23ce5bf85e8b5fc2b6a350/data/virt_autotest/host_unattended_installation_files/combustion/script). I thought this would work on vh015 too. Does it work?

Reply:
It would be better to have this step on any machine. I ever encountered boot order issue, so I added this step.